src-test/org/etools/j1939_84/bus/j1939/packets/SlotTest.java

```diff
@@ -525,4 +524,15 @@ public void testStarDelimitedASCII() {
         assertFalse(slot.isError(data));
     }
 
+    @Test
+    public void verifySpecialCharacters() {
+        assertEquals("km�/h�", Slot.findSlot(469).getUnit());
```

Comment:
Encoding on this review looks ok, but in the IDE (on mac) the superscripts are not printable.

Reply:
Fixed?